src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -24,10 +21,15 @@ export const GithubCreateBranchOptionsGet = async (req: Request, res: Response,
 		return next(new Error(Errors.MISSING_ISSUE_KEY));
 	}
 
+	const jiraHost = jiraHostLocal || getJiraHostFromTenantUrl(tenantUrl);
+
+	// TODO move to middleware or shared for create-branch-get
 	const servers = await getGitHubServers(jiraHost);
 	if (!servers.hasCloudServer && !servers.gheServerInfos.length) {
+		const instance = process.env.INSTANCE_NAME;
 		res.render("no-configuration.hbs", {
-			nonce: res.locals.nonce
+			nonce: res.locals.nonce,
+			configurationUrl: `${jiraHost}/plugins/servlet/ac/com.github.integration.${instance}/github-select-product-page`
```

Comment:
since we no longer exist in the iframe we need to be more explicit with where to go, we dont have AP.navigator access on the FE so we pass the path instead